### ISSUE TITLE
Index value from {#each} seen as missing-declaration

### ISSUE THREAD

**Issue by tivac** [sveltejs/eslint-plugin-svelte3#37]
While I know this keyed `{#each}` is a little silly-looking, it is exposing what I believe is a bug in the plugin.

```html
{#each things as thing, idx (`thing-${idx}`)}
```

is giving me this message:

```
9:63  warning  'idx' is not defined            missing-declaration
```

using svelte@3.5.3, eslint-plugin-svelte3@2.6.0, and eslint@6.0.1

**Comment by Conduitry** [comment]
`missing-declaration` is a warning from the compiler, so this seems like a bug in Svelte and not in this plugin. If you stick the same code in the REPL you'll see that warning there as well.

Does this code work properly apart from the warning? Does the keyed value actually have access to the index variable, or is there a bigger problem here?

**Comment by tivac** [comment]
Whoops, should've dug into that further. The keying is a little nonsensical (keying by only array index is pointless) but does seem to function fine yes.

**Comment by Conduitry** [comment]
Fixed in Svelte v3.6.11.
